### DISCUSSION TITLE
Minimal flash layout for Mender A/B rootfs

### DISCUSSION THREAD

**Discussion by hannes09** [OE4T/meta-tegra#1151] [Ideas]
Hi everyone,
I just watched the discussion form the monthly meeting Jan/23.
Thanks for working with lighting speed on  supporting JP 5.1

I thinks it would also be great if you could provide a minimal partition layout to support mender.
Currently the flash_layou.xml for 4.6 has over 60 partition entrys (spi and sd card)!

I would expect to have only 5 partitions at all : Boot A/B , Rootfs A/B including kernel and dtb files and  a data partition.

**Comment by dwalkes** [comment]
Hi @hannes09 thanks for the feedback.

> I would expect to have only 5 partitions at all : Boot A/B , Rootfs A/B including kernel and dtb files and a data partition.

That would be great however I don't think we can practically do this based on the restrictions we have from the software we get from NVIDIA.  I'm interested to see more about the support for update coming in the March release of Jetpack and interoperability with UEFI.  For now others have successfully made extlinux.conf work with kernel and dtb files, as discussed in the meeting, however I don't think it's possible to remove the dedicated partitions.  I think the remaining partitions are still required by components in the boot chain.
